From dfcb2e59da3e0183eb6ad268e521c3f35fb3e9cf Mon Sep 17 00:00:00 2001
From: Jacob Chen <jacob2.chen@rock-chips.com>
Date: Thu, 5 Jan 2017 15:47:41 +0800
Subject: [PATCH] clk: rockchip: rk3288: make pmu_hclk_otg0 a critical clock

pmu_hclk_otg0 is critical, it will affect usb's access to
the pmu register during reboot.

Change-Id: I3d2b26e10210a91090e3cdee093578a46f6d6ee6
Signed-off-by: Jacob Chen <jacob2.chen@rock-chips.com>
---
 drivers/clk/rockchip/clk-rk3288.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/rockchip/clk-rk3288.c b/drivers/clk/rockchip/clk-rk3288.c
index e8e5f2b2193f..d5989eac06b5 100644
--- a/drivers/clk/rockchip/clk-rk3288.c
+++ b/drivers/clk/rockchip/clk-rk3288.c
@@ -818,6 +818,7 @@ static const char *const rk3288_critical_clocks[] __initconst = {
 	"aclk_vio1_niu",
 	"hclk_peri",
 	"pclk_pd_pmu",
+	"pmu_hclk_otg0",
 };
 
 static void __iomem *rk3288_cru_base;
-- 
2.35.3

